drivers/misc/intel/comm_widget/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+zephyr_library_sources(intel_comm_widget.c)
+zephyr_library_compile_options(-std=gnu99)
```

Comment:
This does not "smell" good, why would such a very generic flag be buried deep down and apply to a single, brand new driver? What problem does this solve?